corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -47,6 +47,7 @@ def render_case(case, options):
     from corehq.apps.hqwebapp.templatetags.proptable_tags import get_tables_as_rows, get_definition
     case = wrapped_case(case)
     timezone = options.get('timezone', pytz.utc)
+    timezone = timezone.localize(datetime.datetime.now()).tzinfo
     _get_tables_as_rows = partial(get_tables_as_rows, timezone=timezone)
```

Comment:
If we have this problem here, don't we have this problem everywhere? Shouldn't this logic be pushed down into the shared code we use for converting timezones? I couldn't tell by just looking, but does `adjust_datetime_to_timezone` for example have this problem, and can it be fixed there?